docker/build_in_docker.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# for error handling:
+set -o pipefail  # trace ERR through pipes
+set -o errtrace  # trace ERR through 'time command' and other functions
+set -o nounset   ## set -u : exit the script if you try to use an uninitialised variable
+set -o errexit   ## set -e : exit the script if any statement returns a non-true return value
+
+
+if [ -z ${IROHA_HOME} ]; then
+    echo "[FATAL] Empty variable IROHA_HOME"
+    exit 1
+fi
+
+
+SOURCE=${IROHA_HOME}
+BUILD=${SOURCE}/build
+TINY=${IROHA_HOME}/docker/tiny
+RELEASE=${TINY}/iroha
+
+
+# run dev container to build iroha
+docker run -i \
+    -v ${IROHA_HOME}/docker/build:/build \
+    -v ${IROHA_HOME}:/opt/iroha \
+    -v ${TINY}:/tiny \
+    -v ${RELEASE}:/release \
+    hyperledger/iroha-dev \
```

Comment:
`warchantua/iroha-dev` is the latest image.
https://github.com/hyperledger/iroha/blob/master/.circleci/config.yml#L7